pkg/values/testdata/values/discovery.yaml

```diff
@@ -27,10 +27,10 @@ pilot:
     topologyKey: "kubernetes.io/hostname"
   podAntiAffinityTermLabelSelector: []
 
-  keepaliveMaxServerConnectionAge: 30m
+  keepaliveMaxServerConnectionAge: 1800s
 
-  deploymentLabels:
-    foo: bar
+#  Fix map[string]interface{}
```

Comment:
Is this a todo?

Reply:
as discussed offline, the values test would be removed